### DISCUSSION TITLE
Anyone still active on this?

### DISCUSSION THREAD

**Discussion by noobishh** [tanjeffreyz/auto-maple#140] [Q&A]
Just found this macro, have many questions. Ive seen people getting success to opening/ using it. Was wondering if anyone out there is nice enough to help please! Thank you!

**Comment by kimjit96** [comment]
hi i am also struggling have you solve the issue?

**Comment by iFernandez96** [comment]
I am a super small time dev and I have developed something pretty similar, but have not posted it on my GitHub. I haven't actually ran the program, but I am willing to try to help if it is within my ability. What problems are you seeing?
